lookup.js

```diff
@@ -171,30 +172,31 @@ exports.init = function (sbot, config) {
 
             sbot.close.hook(function (fn, args) {
               if (liveDrainer) liveDrainer.abort(true)
-              if (removeIndexingActiveListener) removeIndexingActiveListener()
+              if (reindexedDrainer) reindexedDrainer.abort(true)
               if (notifyNewBranch) notifyNewBranch.abort(true)
               fn.apply(this, args)
             })
 
-            // We need to wait for the indexing (including box2 reindexing
-            // encrypted messages) to be done before we can read the messages
-            const obz = sbot.db.getIndexingActive()
-            removeIndexingActiveListener = obz((indexingActive) => {
-              if (indexingActive !== 0) {
-                if (liveDrainer) liveDrainer.abort(true)
-                return
-              }
-
-              pull(
-                sbot.db.query(
-                  where(authorIsBendyButtV1()),
-                  live({ old: true }),
-                  toPullStream()
-                ),
-                pull.filter((msg) => validate.isValid(msg)),
-                (liveDrainer = pull.drain(updateLookupFromMsg))
-              )
-            })
+            pull(
+              sbot.db.reindexed(),
+              pull.filter((msg) => {
+                return (
+                  SSBURI.isBendyButtV1FeedSSBURI(msg.value.author) &&
```

Comment:
We could either remove this check (because validate.isValid checks for msg.value.contentSignature which is duck-typing for bendybutt msgs) OR we could move this check into validate.isValid() itself

Reply:
Okay, deleted the check.